.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+**/logs
```

Comment:
don't forget about empty line in the end of file

Reply:
I'm forget. Sorry